pkg/networkplugin-syncer/handlers/ovn/ovn_logical_router_static_route.go

```diff
@@ -42,7 +42,7 @@ func (ovn *SyncHandler) reconcileSubOvnLogicalRouterStaticRoutes(port, nextHop s
 			return item.OutputPort != nil && *item.OutputPort == port && item.IPPrefix == lrsr.IPPrefix
 		}
 
-		err = libovsdbops.CreateOrUpdateLogicalRouterStaticRoutesWithPredicate(ovn.nbdb, submarinerLogicalRouter, lrsr, LRSRPred)
+		err = libovsdbops.CreateOrReplaceLogicalRouterStaticRouteWithPredicate(ovn.nbdb, submarinerLogicalRouter, lrsr, LRSRPred)
```

Comment:
`CreateOrReplaceLogicalRouterStaticRouteWithPredicate` also removes additional routes. Is  that wanted or relevant in our case? Otherwise we can simply call `CreateOrUpdateLogicalRouterStaticRoutesWithPredicateOps` with nil `[]libovsdb.Operation`.

Reply:
I was following the commit you noted too, but yes, it might not be appropriate.